### PR TITLE
libs: ignore freeRTOS platform

### PR DESCRIPTION
**Pull request by ahaslam2**
After:
`b5692509d tools/scripts: added general makefile`

We get a build error:

```
Compile [  9.3%]: freertos_delay.c
[Fatal Error] freertos_delay.c@40,10: FreeRTOS.h: No such file or directory [ERROR] ./libs/no-OS/drivers/platform/freeRTOS/freertos_delay.c:40:10: fatal error: FreeRTOS.h: No such file or directory
   40 | #include "FreeRTOS.h"
```

Avoid building freeRTOS platform by adding it to the `.mbedigore`

### CODE REVIEW

**Review by mphalke — APPROVED** [review]

**Review by pamolloy — APPROVED** [review]

**Comment by mphalke** [comment]
@ahaslam2 We do not want any commit message with Merge PR# onto main brqanch. Please use rebase (fast-forward) merge strategy going forward or we can do the merge from our side once PR is approved by all reviewers.

**Comment by ahaslam2** [comment]
> @ahaslam2 We do not want any commit message with Merge PR# onto main brqanch. Please use rebase (fast-forward) merge strategy going forward or we can do the merge from our side once PR is approved by all reviewers.

@mphalke, sorry about that. i thought the merge button would do that by default.  I will make sure to select that next time.

**Pull request merged**
